clusters/resource_cluster.go

```diff
@@ -437,6 +445,20 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, c *commo
 		}); err != nil {
 			return err
 		}
+	}
```

Comment:
Can you confirm that library installation on a not-yet-started cluster works?

Reply:
Yes, it does work correctly

Reply:
Could you please check? It's not working for library installation.